lib/Chef/Knife/Cmd.pm

```diff
@@ -51,6 +53,13 @@ Chef::Knife::Cmd - A small wrapper around the Chef 'knife' command line utility
     $knife->vault->update($vault, $item, $values, %options);
     $knife->vault->show($vault, $item_name, %options);
 
+    # knife search commands
+    $knife->search->node($query, %options);
+    $knife->search->client($query, %options);
+
+    # knife data bag commands
+    $knife->data_bag->show($data_bag, %options);
```

Comment:
Usage for this is `knife data bag show BAG [ITEM] (options)`.  I guess `show()` needs to be able handle either of the following:
```
$knife->data_bag->show($data_bag, %options);
$knife->data_bag->show($data_bag, $item, %options);
```

... I guess we can add support for that later though.  I think this is fine for now.